01_todo_app/web/src/components/landing/index.jsx

```diff
@@ -1,68 +1,133 @@
+                            <div>
+                                <p className="text-xs text-yellow-600 mt-1 rounded-md">
+                                    Please be aware that data inside this app is not persisted, and will be lost at the end of the demo!
+                                </p>
+                                <div className="mt-1 flex rounded-md shadow-sm">
+                                    <input
+                                        type="text"
+                                        name="text"
+                                        required
+                                        value={task}
+                                        onChange={(event) => setTask(event.target.value)}
+                                        placeholder="Type your task"
+                                        className="px-6 text-lg py-4 block w-full rounded-none rounded-l-md bg-input text-white placeholder:text-gray-300"
+                                    />
+
+                                    <button
+                                        type="submit"
+                                        className="relative -ml-px inline-flex items-center space-x-2 rounded-r-md text-white text-lg bg-button px-8 md:px-24 py-2 "
+                                    >
+                                        Add
+                                    </button>
+                                </div>
+                            </div>
+
+                            <div>
+                                <input type="hidden" value="NEW" name="status" />
+                            </div>
+                        </form>
+                    </div>
+                    <div className="grid grid-cols-12 mt-20 gap-x-8 md:gap-x-0 gap-y-10">
+                        {messages.map((message) => (
+                            <>
+                                <div className="col-span-1">
+                                    <input
+                                        type="checkbox"
+                                        value=""
+                                        name="checked"
+                                        onChange={(event) => updateTodo(event, message)}
+                                        checked={message.status === "DONE"}
+                                        className="bg-white border border-gray-400 rounded w-6 h-6 text-black p-0 accent-checked"
+                                    />
+                                </div>
+                                <div className="col-span-7 break-words md:col-span-9 text-white">
+                                    <span className={`${message.status == "DONE" ? "line-through" : ""}		text-base`}>{message.text}</span>
+                                </div>
+                                <div className="col-span-1"></div>
+                                <div className="col-span-3 md:col-span-1 flex justify-end">
+                                    <svg
```

Comment:
Exercise to skill-up using components: Extract this to `DeleteIcon` component which should reside in it own file.

In general, please remember about this best practice for future: Each `<svg>` tag responsible for drawing something, should be placed in a isolated component.